src/Loader/loader.scss

```diff
@@ -0,0 +1,106 @@
+@import "~@sebgroup/bootstrap/scss/core";
+@import "~@sebgroup/bootstrap/scss/styles/spinners";
+
+@mixin setLoaderSize($size) {
+    > svg {
+        width: $size;
+        height: $size;
+    }
+}
+
+$offset: 200;
+$duration: 1.5s;
+$duration-sq: 1.2s;
+.rc.loader {
+    color: $green;
+    &:not(.loader-cover):not(.loader-fullscreen) {
+        display: inline-flex;
+        vertical-align: middle;
+    }
+    &.loader-cover,
+    &.loader-fullscreen {
+        left: 0;
+        right: 0;
+        bottom: 0;
+        top: 0;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        z-index: $zindex-tooltip + 10;
```

Comment:
tooltip is already the highest z index value in bootstrap, why the + 10?

Reply:
This is a loader, this makes it higher than a tooltip if it is set to `fullscreen`